edalize/incisive.py

```diff
@@ -0,0 +1,64 @@
+import os
+import logging
+
+from edalize.edatool import Edatool
+
+logger = logging.getLogger(__name__)
+
+
+class Incisive(Edatool):
+
+    _description = """ Cadence Incisive Verilog Backend
```

Comment:
The "Verilog" isn't part of the product name any more I think, it's "Incisive Enterprise Simulator" these days.

Reply:
Ah this was meant to be read as "(Cadence Incisive) (Verilog Backend)" to indicate it's only been tested with verilog and not SV or VHDL